uint8 agentId
uint8 taskId
float64[3] taskLocation
float64 taskDeadline
float64 timestamp
